client/signup/steps/design-picker/index.jsx

```diff
@@ -308,6 +310,7 @@ export default function DesignPickerStep( props ) {
 			stepContent={ renderDesignPicker() }
 			align={ isReskinned ? 'left' : 'center' }
 			skipButtonAlign={ isReskinned ? 'top' : 'bottom' }
+			queryParams={ siteSlugOrId }
```

Comment:
I think we can pass `queryParams` in the following place and we don't need to handle this in every step component
```js
// main.jsx
<CurrentComponent
  ...
  queryParams={ siteIdOrSlug }
/>
```